test/corpus/declarations.txt

```diff
@@ -1149,3 +1149,31 @@ thread_local int c;
   (declaration (storage_class_specifier) (primitive_type) (identifier))
   (declaration (storage_class_specifier) (primitive_type) (identifier))
   (declaration (storage_class_specifier) (primitive_type) (identifier)))
+
+=========================
+Ref-qualifiers
+=========================
+
+void C::f() &;
+void C::f() && {}
```

Comment:
Can you add a couple tests where these declarations and definitions exist inside a class definition?

Reply:
I have done so. In addition, I have added tests that the ref-qualifier interacts properly with other stuff that can be at the end of the function (I have chosen noexcept).